save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt

```diff
@@ -88,44 +82,49 @@ external interface StatusProps<D : Any> : TableProps<D> {
     var status: TestResultStatus?
 
     /**
-     * Name of test suite
+     * File name to filter by
+     */
+    var testName: String?
+
+    /**
+     * Test suite to filer by
      */
     var testSuite: String?
+
+    /**
+     * Tag to filter by
+     */
+    var tag: String?
 }
 
+/**
+ * Test Result Status to filter by
+ * @property status status to filter by
+ * Test Result Status to filter by
+ * @property fileName file name to filter by
+ * File name to filter by
+ * @property testSuite suite to filter by
+ * Test suite to filer by
+ * @property tag tag to filter by
+ * Tag to filter by
+ */
```

Comment:
What is it in kdoc? Why each and every line ends with `to filter by`

Reply:
Done